scrapy/downloadermiddlewares/retry.py

```diff
@@ -70,8 +72,10 @@ def _retry(self, request, reason, spider):
             retryreq.meta['retry_times'] = retries
             retryreq.dont_filter = True
             retryreq.priority = request.priority + self.priority_adjust
+            self.stats.inc_value('retry/count')
```

Comment:
It would tell you explicitly that these exceptions are retried, while `downloader/exception` "only" tells you they happen. Then you may want to disable retries or retry less.